automation/test.sh

```diff
@@ -455,6 +455,10 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} && -z ${label_filter}
   else
     label_filter='(!(Multus,SRIOV,Macvtap,GPU,VGPU,netCustomBindingPlugins))'
   fi
+
+  if [[ ! $TARGET =~ k8s-1.3.* ]]; then
```

Comment:
```suggestion
  if [[ ! $TARGET =~ k8s-1\.3[0-9].* ]]; then
```
It should not happen but the previous regex matches k8s-1.3.0 :)

Reply:
Since this is temporary and the target is managed by us, I don't think we need to be defensive, in worst case it will fail on the test.

Reply:
Sure, thanks!

Reply:
I doubt we are going to change the naming in the future. Most of the time, @dhiller, @brianmcarey  or me are creating the new lanes (and automation). So I can change it but I am pretty confident here.